sdk/python/lib/pulumi/output.py

```diff
@@ -74,20 +74,26 @@ class Output(Generic[T]):
     Future that actually produces the concrete value of this output.
     """
 
-    _resources: Set['Resource']
+    _resources: Awaitable[Set['Resource']]
```

Comment:
Do we have any tests for this in Python?  There were great tests on the TypeScript side - would be nice to have something roughly comparable on the Python side if possible?

Reply:
+1 to this